### PR TITLE
Adds in some job scaling to jobs.txt

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Spawned on MRP as a 4th shaft miner in a random spawn, went to look in jobs.txt and found the settings are utterly insane. The settings for LRP seem to be perfectly normal, but for some reason our MRP job settings are all over the place. (4 shaft miners, 3 geneticists despite 2 computers, 10 security officers roundstart no matter the pop).

Rather than just lower the numbers on some of them, since some of the job slots have wayyyyy too many slots I added the ability for jobs to have population scaling, so there can be less at 0 pop and more at 80 pop.

## Why It's Good For The Game

Adds more jobs with more pop, but still getting a diverse station on low pops.

## Changelog
:cl:
add: Job slots scale
del: Removed sages ability for 4 miners to spawn despite no maps supporting this
del: Removes sages ability for 3 geneticists to spawn despite no maps supporting this
/:cl:

### CODE REVIEW

File: config/Sage/jobs.txt [hunk 1]
```diff
@@ -1,46 +1,59 @@
...
+Chemist=3,2,0,0
+Geneticist=2,2,0,0
+Virologist=1,1,0,0
+
+Scientist=8,6,3,20
+Roboticist=2,2,0,0
+
+Warden=1,1,0,0
+Detective=1,1,0,0
+Security Officer=10,8,2,10
```
Comment (PowerfulBacon) [inline config/Sage/jobs.txt:54]: Populations roundstart:
0: 2 sec
10: 3 sec
20: 4 sec
30: 5 sec
40: 6 sec
50: 7 sec
60: 8 sec

File: config/Sage/jobs.txt [hunk 2]
```diff
@@ -1,46 +1,59 @@
...
+Station Engineer=7,5,2,20
+Atmospheric Technician=4,2,0,0
+
+Medical Doctor=8,6,3,20
+Paramedic=2,1,0,0
+Chemist=3,2,0,0
+Geneticist=2,2,0,0
+Virologist=1,1,0,0
+
+Scientist=8,6,3,20
```
Comment (PowerfulBacon) [inline config/Sage/jobs.txt:49]: Populations roundstart:
0: 3
20: 4
40: 5
60: 6

File: config/Sage/jobs.txt [hunk 3]
```diff
@@ -1,46 +1,59 @@
...
+Mime=1,1,0,0
+Curator=1,1,0,0
+Lawyer=2,2,0,0
+
+Chaplain=1,1,0,0
+
+Station Engineer=7,5,2,20
+Atmospheric Technician=4,2,0,0
+
+Medical Doctor=8,6,3,20
```
Comment (PowerfulBacon) [inline config/Sage/jobs.txt:43]: Populations roundstart:
0:3
20:4
40:5
60:6

File: config/Sage/jobs.txt [hunk 4]
```diff
@@ -1,46 +1,59 @@
...
+
+Clown=1,1,0,0
+Mime=1,1,0,0
+Curator=1,1,0,0
+Lawyer=2,2,0,0
+
+Chaplain=1,1,0,0
+
+Station Engineer=7,5,2,20
+Atmospheric Technician=4,2,0,0
```
Comment (PowerfulBacon) [inline config/Sage/jobs.txt]: 4 atmos techs lol? this should be changed

File: config/Sage/jobs.txt [hunk 5]
```diff
@@ -1,46 +1,59 @@
...
+Janitor=3,2,0,0
+
+Clown=1,1,0,0
+Mime=1,1,0,0
+Curator=1,1,0,0
+Lawyer=2,2,0,0
+
+Chaplain=1,1,0,0
+
+Station Engineer=7,5,2,20
```
Comment (PowerfulBacon) [inline config/Sage/jobs.txt:40]: Populations roundstart:
0:2
20:3
40:4
60:5

**Comment by ike709** [comment]
It predates the popcap, when we had to figure out how to prevent 100 assistants from existing.

**Comment by ghost** [comment]
I added this :)

**Comment by ghost** [comment]
>sages ability for 4 miners to spawn despite no maps supporting this

This is actually wrong, the mining station has another 3 equipment lockers, so you could easily fit 6 miners.

**Comment by PowerfulBacon** [comment]
> 
> 
> > sages ability for 4 miners to spawn despite no maps supporting this
> 
> This is actually wrong, the mining station has another 3 equipment lockers, so you could easily fit 6 miners.

The mining station has 3 mining suits which are essentially required to mine. Also almost no maps support 3 miners and you get placed in a random spawn.
This happened to me on MRP, made it essentially so I couldn't mine so I just ran into lava and DNRed

File: config/Sage/jobs.txt (hunk 1 above)
Comment (Archanial) [reply config/Sage/jobs.txt:54]: 7-8 is too much in my opinion, considering there is HoS, warden, detective and brig physician.

**Pull request closed without merging**